### PR TITLE
DS buff

### PR DESCRIPTION
**Pull request by k-k-karma**
## Changelog
:cl: karma

balance: DS ARE UNSTUNNABLE AND IGNORE SLOWDOWN AND HAVE NO DROP SHIELDS AS INTENDED
/:cl:

### CODE REVIEW

File: hippiestation/code/game/objects/items/shields.dm [hunk 1]
```diff
@@ -21,4 +21,10 @@
 		if (prob(final_block_chance))
 			owner.visible_message("<span class='danger'>[owner] skillfully blocks [attack_text] with [src]!</span>")
 			return TRUE
-	return FALSE
\ No newline at end of file
+	return FALSE
+/obj/item/shield/energy/attack_self(mob/living/carbon/human/user)
+	. = ..()
+	if(active)
+		src.add_trait(TRAIT_NODROP, CLOTHING_TRAIT)
```
Comment (yoyobatty) [inline hippiestation/code/game/objects/items/shields.dm:28]: Explicit src?

File: hippiestation/code/game/objects/items/shields.dm [hunk 2]
```diff
@@ -21,4 +21,10 @@
...
 			return TRUE
-	return FALSE
\ No newline at end of file
+	return FALSE
+/obj/item/shield/energy/attack_self(mob/living/carbon/human/user)
+	. = ..()
+	if(active)
+		src.add_trait(TRAIT_NODROP, CLOTHING_TRAIT)
+	else
+		src.remove_trait(TRAIT_NODROP, CLOTHING_TRAIT)
```
Comment (yoyobatty) [inline hippiestation/code/game/objects/items/shields.dm:30]: Explicit src?

**Comment by Jujumatic** [comment]
might as well give them adminordrazine at this point. fuck this

**Comment by WarRage333** [comment]
this is dumb

**Comment by TheMeatGod** [comment]
-1

**Comment by Carbonhell** [comment]
this is getting too far, -1

**Comment by DeltaFire15** [comment]
DS already DABS on the station, give the ultimate robustos a chance to beat em.

**Comment by DaydreamPuppeh** [comment]
Karma, both of your recent prs have been shitposts. Why

**Comment by k-k-karma** [comment]
All of these were in the original PR, but were merged w/o working

**Comment by uBadush** [comment]
stunimmune plus no item drop, no thats gay as fuck

**Comment by Jujumatic** [comment]
@k-k-karma then revert them because as you can see we don't want it reeeeeee

**Comment by Lxtoofonea** [comment]
-1 from me

File: hippiestation/code/game/objects/items/shields.dm (hunk 1 above)
Comment (k-k-karma) [reply hippiestation/code/game/objects/items/shields.dm]: yes

File: hippiestation/code/game/objects/items/shields.dm (hunk 2 above)
Comment (k-k-karma) [reply hippiestation/code/game/objects/items/shields.dm]: yes

**Comment by Jujumatic** [comment]
Bye bye sweet prince

**Comment by k-k-karma** [comment]
this is a fix

**Pull request closed without merging**